server-src/Main.hs

```diff
@@ -88,45 +88,44 @@ app cache request respond = do
     (requestMethod request <> " " <> rawPathInfo request)
 
   case request of
-    GET [] -> do
-      indexHtml <- getDataFileName "index.html"
-      respond
-        (responseFile
-          status200
-          [(hContentType, "text/html")]
-          indexHtml
-          Nothing)
 
-    GET ["branch", branch] ->
+    GET ["api", "branch", branch] ->
       handleGetBranch cache branch >>= respond
 
-    GET ["declaration", declaration] ->
+
+    GET ["api", "declaration", declaration] ->
       handleGetDeclaration cache declaration >>= respond
 
-    GET ["head"] -> do
+    GET ["api", "head"] -> do
       [head] <- listDirectory ".unison/v1/paths/_head"
       respond
         (responseLBS
           status200
           [(hContentType, "application/json")]
           (LazyByteString.Char8.pack (show head)))
 
-    GET ["patch", patch] ->
+    GET ["api", "patch", patch] ->
       respond
         (responseFile
           status200
           [(hContentType, "application/octet-stream")]
           (".unison/v1/patches/" <> Text.unpack patch <> ".up")
           Nothing)
 
-    GET ["term", term, "term"] ->
+    GET ["api", "term", term, "term"] ->
       handleGetTerm cache term >>= respond
 
-    GET ["term", term, "type"] ->
-      handleGetTermType cache term >>= respond
+    GET ["api", "term", termType, "type"] ->
```

Comment:
`term` is the right variable name here, the slug is the hash of the term, not its type.

Reply:
:grimacing: Gotcha, will fix in a minute here. I think that answers one question I had with the term type route also, I might check in with you about that before I send the next PR.